settings.gradle

```diff
@@ -1,3 +1,17 @@
+pluginManagement {
```

Comment:
This block is required to be able to resolve plugins in the root build.gradle plugins block.

Reply:
curious: does gradle sync fail without this block?

Reply:
Yep there were failures due to this plugins block in the root build.gradle file:

```groovy
plugins {
    alias libs.plugins.dokka
    alias libs.plugins.gradle.nexus.publish
    alias libs.plugins.detekt

    alias libs.plugins.android.application apply false
    alias libs.plugins.android.library apply false
    alias libs.plugins.kotlin.android apply false
    alias libs.plugins.kotlin.parcelize apply false
    alias libs.plugins.kotlin.kapt apply false
    alias libs.plugins.androidx.safeargs apply false
}
```